test/e2e-go/restAPI/restClient_test.go

```diff
@@ -170,14 +170,14 @@ func waitForTransaction(t *testing.T, testClient libgoal.Client, fromAddress, tx
 	}
 	timeoutTime := time.Now().Add(timeout)
 	for {
-		tx, err = testClient.TransactionInformation(fromAddress, txID)
-		if err != nil && strings.HasPrefix(err.Error(), "HTTP 404") {
-			tx, err = testClient.PendingTransactionInformation(txID)
-		}
+		// tx, err = testClient.TransactionInformation(fromAddress, txID)
```

Comment:
@algochoi Sanity checking:  Are these lines commented out because v2 offers no analogous API?

If so, do you agree with the following statement?

It feels like an analogous v2 API is needed or else tests are liable to execute non-deterministically.

Reply:
Upon further review, I see the v2 endpoint implementation looks for confirmed transactions when no pending transactions are found.

My updated view is (that I'd like you to confirm is accurate):  The commented out lines can be safely removed.

Reply:
Sorry - My above comment conflates behavior with `PendingTransactionInformation`.  I'm back to my original thread remarks.

Reply:
@algochoi Resolving - I agree with your assessment.  Thanks for the change.